Lib/test/test_tools/test_i18n.py

```diff
@@ -413,6 +413,12 @@ def test_files_list(self):
             self.assertIn(f'msgid "{text2}"', data)
             self.assertNotIn(text3, data)
 
+    def test_help_text(self):
+        """Test that the help text is displayed."""
+        res = assert_python_ok(self.script, '--help')
+        self.assertEqual(res.out, b'')
+        self.assertIn(b'pygettext -- Python equivalent of xgettext(1)', res.err)
```

Comment:
I only test that the `--help` argument works, not that the help text matches a snapshot exactly, though I can add it